### ISSUE TITLE
x360ce no longer initializing on RocketLeague startup.

### ISSUE THREAD

**Issue by Joooop** [x360ce/x360ce#250]
Hello,

I've been using x360ce to use my gamecube controllers for about 3 months now. They've always worked. Yesterday I tried to play some rocket league and I did not get an init beep on startup and my inputs are no longer being registered.

I've tried ports 1-4, none work. I've also tried another game that I have x360ce setup on and my controllers registered fine there. 

The only thing I can think of that happened right before this stopped working was downloading rocket league on another machine under the same steam account. Did a Steam cloud sync change something?

Any help/possible fixes are appreciated!

**Comment by Squall-Leonhart** [comment]
did you turn gamepad support off on the other machine? settings are synched.

**Comment by Joooop** [comment]
So after this initially happened, I uninstalled the game from my second machine. I just re-installed it and tried using my controller on the second machine. Everything worked fine. 

I then went to my main system, synced settings from the cloud and then tried my controller. It still did not fix the issue so I'm currently doing a fresh re-install on my main system. 

If this doesn't work I'm going to try and manually sync the ini files / settings as I could not find any specific gamepad toggle in the UI. Nor could I find anything from browsing the config files.
